saleor/dashboard/graphql/api.py

```diff
@@ -4,25 +4,34 @@
 
 from ...graphql.core.filters import DistinctFilterSet
 from ...graphql.page.types import Page
-from ...graphql.product.types import Category, resolve_categories
+from ...graphql.product.types import Category, Product, resolve_categories
 from ...graphql.utils import get_node
+from ...product import models as product_models
 from .page.mutations import PageCreate, PageDelete, PageUpdate
 from .page.types import resolve_all_pages
+from .product.filters import ProductFilter
 from .product.mutations import (
     CategoryCreateMutation, CategoryDelete, CategoryUpdateMutation)
 
 
 class Query(graphene.ObjectType):
+    node = graphene.Node.Field()
+    debug = graphene.Field(DjangoDebug, name='__debug')
+
     categories = DjangoFilterConnectionField(
         Category, filterset_class=DistinctFilterSet,
         level=graphene.Argument(graphene.Int))
     category = graphene.Field(Category, id=graphene.Argument(graphene.ID))
-    node = graphene.Node.Field()
-    debug = graphene.Field(DjangoDebug, name='__debug')
+
     page = graphene.Field(Page, id=graphene.Argument(graphene.ID))
     pages = DjangoFilterConnectionField(
         Page, filterset_class=DistinctFilterSet)
 
+    product = graphene.Field(Product, id=graphene.Argument(graphene.ID))
+    products = DjangoFilterConnectionField(
+        Product, filterset_class=ProductFilter,
+        categoryID=graphene.Argument(graphene.ID))
```

Comment:
We should use *snake_case* consistently on Python side, so instead of `categoryID` it should be `category_id`. This will be nicely translated to `categoryId` in GraphQL.